src/VSSetup.PowerShell/VSSetup.PowerShell.csproj

```diff
@@ -41,10 +42,10 @@
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.PowerShell.3.ReferenceAssemblies.1.0.0\lib\net4\System.Management.Automation.dll</HintPath>
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\lib\PowerShell\System.Management.Automation.dll</HintPath>
```

Comment:
Ick. While you are taking care of the CLA requirement, let me talk with the PowerShell team and see what we can do about this. I'd rather not check in any binaries files, though at least this won't change. Perhaps I can get the asmmeta version (no IL; smaller).